pkg/commands/git_commands/commit_loader.go

```diff
@@ -349,12 +349,25 @@ func (self *CommitLoader) setCommitMergedStatuses(refName string, commits []*mod
 	if ancestor == "" {
 		return commits
 	}
+	inMainBranch := false
+	branchRef, err := self.cmd.New(
+		NewGitCmd("symbolic-ref").Arg("HEAD").
+			ToArgv(),
+	).DontLog().RunWithOutput()
+	if err == nil {
+		branchRef = strings.TrimSpace(branchRef)
+		for _, mainRef := range self.mainBranches {
+			if mainRef == branchRef {
+				inMainBranch = true
+			}
+		}
```

Comment:
This loop can be shortened to a one-liner by saying `inMainBranch = lo.Contains(self.mainBranches, branchRef)`.

BTW, I'd prefer the name `isMainBranch`, but that's a nitpick.